src/style.css

```diff
@@ -0,0 +1,21 @@
+body {
+  margin: 0;
+}
+
+#red {
+  background-color: red;
+  height: 100px;
+  width: 100%;
+}
+
+#green {
+  background-color: green;
+  height: 100px;
+  width: 100%;
+}
+
+#blue {
+  background-color: blue;
+  height: 100px;
+  width: 100%;
+}
```

Comment:
If we have 5 blocks and these blocks have common characteristics, such as height, width, color, fonts, etc., then we should move these characteristics in a separate class and use it for style these blocks. DRY - Don't Repeat Yourself.

Reply:
IDs are used for JS, try not to style the id if you can avoid it. This makes impossible to reuse the code. Stylize classes. [Style guide](https://mate-academy.github.io/style-guides/htmlcss.html#id-selectors)

Reply:
The name of the element class should make a sense. It should describe what the role of the element is on this page or describe its state. In the case with the name _red_, _green_, maybe in the future, the color of your block will be changed, so you have to rename all classes on the page.